### PR TITLE
Keyboard Press Feature and Version String Updated

### PR DESCRIPTION
**Pull request by ykeara**
### New Feature
This Feature adds 2 new actions that behave like Keys on keyboard to better allow features such as PTT in third party programs such as discord.

By Default the keys are both set to F9

We use the same token parser, we just ONLY Take the first token, and send a key press down event and up event discreetly.

The new feature will ignore longer strings i.e. ^abc will only send teh ^ key.

any key supported by keyboard spec will work for our situation as well.

### New Actions:

Keyboard Press One - Action to press a single key.
Keyboard Press Two

### CODE REVIEW

File: src/keyboard_input/input_sender.h [hunk 1]
```diff
@@ -75,6 +75,26 @@ inline void sendTokensAsInput( const std::vector<Token> tokens )
...
+    initOsSystems();
+
+    sendKeyPress( token, event );
+
+    shutdownOsSystems();
+}
+
+inline void sendFirstCharAsInput( const std::string inputstring,
+                                  KeyStatus event )
+{
```
Comment (username223) [inline src/keyboard_input/input_sender.h:89]: You'll need calls to `initOsSystems()` and `shutdownOsSystems()` in this function like `sendTokenPress` above.

Comment (username223) [reply src/keyboard_input/input_sender.h:89]: Nvm, read wrong. ;)

**Comment by username223** [comment]
Any public documentation for the new feature?

**Comment by ykeara** [comment]
None Yet, It Mirrors current system just only parses/handles 1 token. Closer to Release Window I would like to spend some time updating all documentation. 

with this release I would like to re-emphasize current capabilities in addition to new ones.... so Kinda want to increase visibility of current available documentation, but haven't really gotten around to examining it yet.

**Pull request merged**